src/settings.cpp

```diff
@@ -533,6 +533,13 @@ if ( GetFlagIniSet ( IniXMLDocument, "client", "defcentservaddr", bValue ) )
         bWindowWasShownConnect = bValue;
     }
 
+    // selected Settings Tab
+    if ( GetNumericIniSet ( IniXMLDocument, "client", "settingstab",
```

Comment:
clang-format will have line width somewhere around the 150 mark, so some lines are wrapped a bit narrow.